app/src/main/java/pl/gittobefit/WorkoutDisplay/dialog/EditExerciseDialog.java

```diff
@@ -0,0 +1,141 @@
+package pl.gittobefit.WorkoutDisplay.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.NumberPicker;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatDialogFragment;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.navigation.Navigation;
+
+import java.util.ArrayList;
+
+import pl.gittobefit.R;
+import pl.gittobefit.WorkoutDisplay.objects.ExerciseExecution;
+import pl.gittobefit.WorkoutDisplay.viewmodel.InitiationTrainingDisplayLayoutViewModel;
+import pl.gittobefit.database.AppDataBase;
+import pl.gittobefit.database.pojo.ExerciseExecutionPOJODB;
+
+public class EditExerciseDialog extends AppCompatDialogFragment implements NumberPicker.OnValueChangeListener
+{
+    private View myView;
+    private NumberPicker seriesNumberPicker;
+    private NumberPicker countNumberPicker;
+    private int position;
+    private String scheduleType;
+    private ArrayList<ExerciseExecutionPOJODB> exercisesExecutionArrayList;
+    private int trainingID;
+    private String exerciseName;
+    private ArrayList<ArrayList<ExerciseExecutionPOJODB>> exerciseExecutionPOJODBS;
+
+    public EditExerciseDialog(View view, String scheduleType, int position, ArrayList<ExerciseExecutionPOJODB> exercisesExecutionArrayList,
+                              int trainingID, String exerciseName, ArrayList<ArrayList<ExerciseExecutionPOJODB>> exerciseExecutionPOJODBS)
+    {
+        this.myView = view;
+        this.scheduleType = scheduleType;
+        this.position = position;
+        this.exercisesExecutionArrayList = exercisesExecutionArrayList;
+        this.trainingID = trainingID;
+        this.exerciseName = exerciseName;
+        this.exerciseExecutionPOJODBS = exerciseExecutionPOJODBS;
+    }
+
+    @NonNull
+    @Override
+    public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+        LayoutInflater inflater = getActivity().getLayoutInflater();
+        View view = inflater.inflate(R.layout.layout_edit_exercise_dialog, null);
+
+        seriesNumberPicker = view.findViewById(R.id.seriesNumberPicker);
+        seriesNumberPicker.setMaxValue(10);
+        seriesNumberPicker.setValue(exercisesExecutionArrayList.get(position).getSeries());
+        seriesNumberPicker.setMinValue(1);
+        seriesNumberPicker.setWrapSelectorWheel(false);
+        seriesNumberPicker.setOnValueChangedListener(this);
+
+        countNumberPicker = view.findViewById(R.id.countNumberPicker);
+
+        if (exercisesExecutionArrayList.get(position).getTime() > 20) {
+            countNumberPicker.setMaxValue(90);
+            countNumberPicker.setMinValue(10);
```

Comment:
warunek chyba nie taki sam tutaj pozwalasz zeby cwiczenie trwalo 10 sekund a sprawdzasz dla 20 
nwm tez czy 90 sekund to nie za mało mogą być cwiczenia jak bieg na biezni ktore ktos moglby chciec robic dłuzej ale to zobaczymy co chlopaki powiedza